ldap-user-cleanup/src/main/resources/Scheduler/LDAP/Translations.xml

```diff
@@ -63,7 +66,9 @@ org.xwiki.contrib.ldap.scheduler.LDAP.OldUserCleanupJob.replaceAuthor.hint=Set t
 org.xwiki.contrib.ldap.scheduler.LDAP.OldUserCleanupJob.replaceContentAuthor.hint=Set this option if deleted users should be replaced as the last content author of pages.
 org.xwiki.contrib.ldap.scheduler.LDAP.OldUserCleanupJob.replaceCommentAuthor.hint=Set this option if deleted users should be replaced as the creator of comments.
 org.xwiki.contrib.ldap.scheduler.LDAP.OldUserCleanupJob.dryRun.hint=Set this option if you want the job to make no actual changes but only log any actions that would be performed.
-
+org.xwiki.contrib.ldap.scheduler.LDAP.OldUserCleanupJob.shouldDisable.hint=Set this option if you want the users to be disabled rather than deleted.
+org.xwiki.contrib.ldap.scheduler.LDAP.OldUserCleanupJob.activeAttribute.hint=The LDAP attribute that conveys whether an user is active or not. For active directory, the attribute is 'userAccountControl'.
```

Comment:
Nitpick: should we write "active directory" (lowercase) or "Active Directory" (capitalized) (see next translation)? I tend to choose the latter.
Also maybe (but just maybe) begin the sentence with an "E.g. for Active Directory ..." so other LDAP schemes do not feel excluded ;-)  (Then I do not know any other LDAP scheme that has an "disabled" flag for user accounts. Maybe Apple Open Directory?)

Reply:
Edited the translations as per your observations. 

I personally didn't know any LDAP schema other than `Active Directory` :stuck_out_tongue: . But after some research, it seems that `Red Hat Directory Server` has an attribute to mark an user as inactive. `Apple Open Directory` doesn't 🤔.